### PR TITLE
fixes blobs lighting cyborgs on fire

### PR DESCRIPTION
**Pull request by Iamgoofball**
fuq

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm [hunk 1]
```diff
@@ -52,8 +53,9 @@ datum/reagent/blob/skin_melter/reaction_mob(var/mob/living/M as mob, var/method=
 	if(method == TOUCH)
 		M.apply_damage(10, BRUTE)
 		M.apply_damage(10, BURN)
-		M.adjust_fire_stacks(2)
-		M.IgniteMob()
+		if(iscarbon(M))
+			M.adjust_fire_stacks(2)
+			M.IgniteMob()
 		if(show_message)
 			M << "<span class = 'userdanger'>The blob strikes you, and you feel your skin char and melt!</span>"
```
Comment (RandomMarine) [inline code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm]: While you're touching this file. Can you do something about #8397?

**Comment by Jordie0608** [comment]
You need to mention this fixes #8471

**Comment by RemieRichards** [comment]
Why? borgs have a ```handle_fire()```, they should be able to be set on fire.

**Comment by dannno** [comment]
^ yeah
lets borgs be set on fire but not take incredible damage from it

>AI malf,  floods plasma
>station in chaos
>survivors trying to crowbar past a huge fire
>flaming borg rolls up through the flames
>slaughters them all

shit sounds fantastic

**Comment by Iamgoofball** [comment]
not a bug

**Pull request closed without merging**